tensorflow/contrib/makefile/Makefile

```diff
@@ -910,6 +910,12 @@ $(PROTO_TEXT_OBJS) : $(PROTO_TEXT_PB_H_FILES)
 
 # Ensures we link CoreFoundation as it is used for time library when building
 # for Mac.
+ifeq ($(TARGET),OSX)
+  ifeq($(HOST_ARCH), x86_64)
+	HOST_LDOPTS += -framework CoreFoundation
```

Comment:
Just realized you're mixing tabs and spaces here

Reply:
Fixed.